### PR TITLE
oob: Always run enterprise OOB migrations in worker again

### PR DESCRIPTION
**Pull request by eseliger**
In https://sourcegraph.sourcegraph.com/github.com/sourcegraph/sourcegraph/-/commit/591d702d290c5a998bc08c4e4e66c8914407d542, we moved this init script from enterprisecmd to frontend, which disabled the enterprise migrations to be considered in other services - namely worker which is running the migrations in the background for active instances.

That means that we haven't been running a few migrations while instances are running in 5.1, but that shouldn't be a big problem luckily because
- none are deprecated in 5.1
- none were introduced in 5.1
- only a few from 5.0 would potentially need to run
- but they likely already completed while the instance was on 5.0, unless there was a lot of stuff to be migrated
- when an instance first booted on 5.1, these would have never run, but also no old data would exist

## Test plan

Verified migrations run again.

### CODE REVIEW

File: enterprise/cmd/sourcegraph/enterprisecmd/enterprisecmd.go [hunk 1]
```diff
@@ -4,10 +4,16 @@
 package enterprisecmd
 
 import (
+	"github.com/sourcegraph/sourcegraph/internal/oobmigration"
 	"github.com/sourcegraph/sourcegraph/internal/service"
 	"github.com/sourcegraph/sourcegraph/internal/service/svcmain"
 )
 
+func init() {
+	// TODO(sqs): TODO(single-binary): could we move this out of init?
```
Comment (camdencheek) [inline enterprise/cmd/sourcegraph/enterprisecmd/enterprisecmd.go:13]: gotta love TODOs with foresight

**Review by camdencheek — APPROVED** [review]

**Review by efritz — APPROVED** [review]

**Review by sashaostrikov — APPROVED** [review]

**Comment by eseliger** [comment]
@efritz @camdencheek @sashaostrikov I realized that this broke it on the frontend side now, because that one does not actually use enterprisecmd (don't ask me why). I went ahead and fixed that by removing the enterprise distinction entirely - we don't need it anymore in our all enterprise repo anyways.

**Review by efritz — APPROVED** [review]
All enterprise!

**Review by sashaostrikov — APPROVED** [review]
Nice, LGTM!

**Pull request merged**